po/README.md

```diff
@@ -1,14 +1,15 @@
 # i18n
 
-Translations are not implemented yet, see github issue #728. Please read the notes before opening a PR.
+i18n build only works with the meson build system. See github issue #728 for more details.
```

Comment:
Honestly, if this will be merged, I will open a PR to default to Meson (change is fine I just wanted to mention it).

Reply:
I rolled back the README changes

Reply:
I rolled back `README.md` too when I rolled back `autogen.sh` changes. You can change it yourself later. 😃